setup.py

```diff
@@ -205,6 +205,9 @@ def _download_and_extract_local_driver(
     long_description=Path("README.md").read_text(encoding="utf-8"),
     long_description_content_type="text/markdown",
     url="https://github.com/Microsoft/playwright-python",
+    project_urls={
+        "Release notes": "https://playwright.dev/python/docs/release-notes",
```

Comment:
```suggestion
        "Release notes": "https://github.com/microsoft/playwright-python/releases",
```

Reply:
personally I'd prefer to read the release notes within the docs site, so they are formatted the same as the rest of the docs that they link to

Reply:
We were thinking about stopping mirroring the release-notes in the docs, since they are already on GitHub and having them in the docs is a manual effort. Or at least having them mirrored into playwright.dev.

Reply:
okay, well you do you, you're free to push to my PR